crates/catalog/rest/src/catalog.rs

```diff
@@ -103,8 +104,7 @@ impl RestCatalogConfig {
         ])
     }
 
-    fn try_create_rest_client(&self) -> Result<HttpClient> {
-        // TODO: We will add ssl config, sigv4 later
+    fn get_default_headers(&self) -> Result<HeaderMap> {
```

Comment:
Minor: we may simplify the name to `headers()` or `http_headers()`, this is consistent with other function name like `table_endpoint`, `url_prefixed`, etc.

Reply:
+1

Reply:
cc @whynick1 Would you mind to do this rename in this pr or in following prs?

Reply:
Rename it `http_headers()`. 9ab76786ce65c73f6ee691a5542e57a54bdc420e